src/EditorFeatures/CSharpTest/ReplaceDocCommentTextWithTag/ReplaceDocCommentTextWithTagTests.cs

```diff
@@ -448,6 +448,42 @@ public async Task TestInXMLAttribute2()
 class C
 {
     void WriteLine<TKey>(TKey value) { }
+}");
+        }
+
+        [WorkItem(38370, "https://github.com/dotnet/roslyn/issues/38370")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceDocCommentTextWithTag)]
+        public async Task TextBaseKeyword()
```

Comment:
Not sure if this is or not a typo, maybe use `Test` instead of `Text` ?

Reply:
Typo :( Thank you!!